geo/src/algorithm/affine_ops.rs

```diff
@@ -115,6 +115,19 @@ impl<T: CoordNum, M: MapCoordsInPlace<T> + MapCoords<T, T, Output = Self>> Affin
 ///     (x: -0.5688687, y: 5.5688687)
 /// ], max_relative = 1.0);
 /// ```
+///
+/// ## Create affine transform manually, and access elements using getter methods
+/// ```
+/// let transform = AffineTransform::new(10.0, 0.0, 400_000.0, 0.0, -10.0, 500_000.0);
+///
+/// let a: f64 = transform.a();
+/// let b: f64 = transform.b();
+/// let c: f64 = transform.xoff();
```

Comment:
Maybe name this variable binding `xoff`? Not sure

Reply:
Yes, `xoff` please.

Reply:
Done in a6138f18791d079029fb59b453506f3f754c57bd